lib/portlayer/exec/commit.go

```diff
@@ -246,6 +216,94 @@ func Commit(op trace.Operation, sess *session.Session, h *Handle, waitTime *int3
 	return nil
 }
 
+// batchBlockOnFunc is a batching routine that batch processes incoming requests.
+// Incoming request signals the batching routine by throwing into a batch channel. When this routine is performing operation on
+// the previous requests, new requests will be batched in the channel, waiting for next iteration to process.
+func batchBlockOnFunc(ctx context.Context, batch chan chan error, operation func(operation trace.Operation) error) {
+	op := trace.FromContext(ctx, "Add container VM to VMGroup dispatch routine")
```

Comment:
technically this label is only correct given the `operation` function that is being passed in, but that's just an FYI comment and not a request for update.

Reply:
Yea, the context label here is not a generic one. Easier for user to understand when looking at the log I guess